servers/src/main/java/tachyon/master/permission/FileSystemPermissionChecker.java

```diff
@@ -34,6 +39,9 @@
   /** The super group of Tachyon file system. All users in this group have super permission. */
   private static String sFileSystemSuperGroup;
 
+  /** Used to traverse a path and get all inodes on the path. */
+  private static InodeTree sInodeTree;
```

Comment:
Would it better not expose InodeTree, but get the `inodes` from the FileSystemMaster?